doc/source/user_guide/missing_data.rst

```diff
@@ -239,6 +239,16 @@ of ways, which we illustrate:
    df2.fillna(0)
    df2["one"].fillna("missing")
 
+**Replace NA for columns of specific data types**
+
+.. ipython:: python
+
+nonDateDf = df.select_dtypes(exclude=['datetime'])
```

Comment:
need to be indented. don't use camelCase, pref `non_date_df`

Reply:
its not clear what you are trying to do here, can you add a comment & show the resulting frame (and the before one if needed); you are assigning in place so it won't show currently.